src/v0/util/facebookUtils/networkHandler.js

```diff
@@ -151,6 +151,47 @@ const errorDetailsMap = {
       .setMessage('API User Too Many Calls')
       .build(),
   },
```

Comment:
As the error codes handling is common across fb destinations, do the codes have same meaning/purpose across fb destinations ?

Reply:
yes. It is. Or we can say that, none of a single error code has different meaning in different destinations.